nf_core/components/install.py

```diff
@@ -264,3 +267,25 @@ def clean_modules_json(self, component, modules_repo, modules_json):
                             self.component_type, component, repo_to_remove, modules_repo.repo_path
                         )
                         return component_values["installed_by"]
+
+    def check_alternate_remotes(self, modules_json):
+        """
+        Check whether there are previously installed components with the same org_path but different remote urls
+        Log warning if multiple remotes exist.
+
+        Return:
+            True: if problematic components are found
+            False: if problematic components are not found
+        """
+        alternate_remotes = False
+        modules_json.load()
+        for repo_url, repo_content in modules_json.modules_json.get("repos", dict()).items():
+            for dir in repo_content.get(self.component_type, dict()).keys():
+                if dir == self.org and repo_url != self.modules_repo.remote_url:
+                    alternate_remotes = True
+        if alternate_remotes:
+            warn_msg = f"Multiple module remotes are used with the same org_path '{self.org}': {', '.join(alternate_remotes)}. This may result in reinstalled modules from the wrong remote."
```

Comment:
I would move this warning where you run the function. And use `log.error()` instead of warning as I said in the comment above.
`alternate_remotes` is a boolean, I think it will be best to reword the message a little bit, similar to:
```suggestion
            warn_msg = f"You are trying to install modules from a different repositories with the same organisation name '{self.org}' (set in the `.nf-core.yml` file under `self_org`).\nThis is not supported, and will likely cause problems, self_org should be set to the github account/organisation name."
```

Reply:
i've updated the message and turned it into an error, and moved it up to where the function is run.